conf/config-default.yaml

```diff
@@ -397,6 +397,7 @@ plugins:                          # plugin list (sorted by priority)
   - openwhisk                      # priority: -1901
   - serverless-post-function       # priority: -2000
   - ext-plugin-post-req            # priority: -3000
+  #- ext-plugin-post-resp          # priority: -4000
```

Comment:
We can enable it by default?

Reply:
All back-to-origin logic will be hooked by `resty.http`. So I think it is not appropriate to enable this plugin by default until the relevant logic is perfect.

Reply:
Only route that contains this plugin should use `resty.http` instead.